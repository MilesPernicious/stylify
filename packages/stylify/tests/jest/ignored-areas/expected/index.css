.a{
	color: red
}
.b{
	color: darkorange
}
.c{
	color: black
}
.d{
	color: darkblue
}
.e{
	color: brown
}
