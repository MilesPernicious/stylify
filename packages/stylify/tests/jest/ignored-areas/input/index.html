<div class="color:red"></div>

<code-element>
	<div class="color:darkorange"></div>
	<!-- This single closed element here is correct -->
	</code>
	<code class="color:black">
		<!-- stylify-ignore -->
			<div class="color:darkgreen"></div>
		<!-- /stylify-ignore -->
	</code>
	<div class="color:darkblue"></div>
</code-element>

<!-- This tests if previous code-element is not wrongly matched (closed too early) -->
<div class="color:brown"></div>

<!-- stylify-ignore -->
<div class="color:green"></div>
<!-- /stylify-ignore -->

<code>
	<div class="color:blue"></div>
</code>

<pre>
	<div class="color:yellow"></div>
</pre>

<head>
	<div class="color:pink"></div>
</head>

<script>
	<div class="color:orange"></div>
</script>

