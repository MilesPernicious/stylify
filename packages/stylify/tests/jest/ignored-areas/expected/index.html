<div class="_9kvng"></div>

<code-element>
	<div class="_aihf5"></div>
	<!-- This single closed element here is correct -->
	</code>
	<code class="_oefhb2">
		<!-- stylify-ignore -->
			<div class="color:darkgreen"></div>
		<!-- /stylify-ignore -->
	</code>
	<div class="_228bhf"></div>
</code-element>

<!-- This tests if previous code-element is not wrongly matched (closed too early) -->
<div class="_oe9l1b"></div>

<!-- stylify-ignore -->
<div class="color:green"></div>
<!-- /stylify-ignore -->

<code>
	<div class="color:blue"></div>
</code>

<pre>
	<div class="color:yellow"></div>
</pre>

<head>
	<div class="color:pink"></div>
</head>

<script>
	<div class="color:orange"></div>
</script>

